src/core/search/indices.cc

```diff
@@ -35,6 +35,18 @@ vector<string> GetWords(string_view text) {
   return vector<string>{make_move_iterator(words.begin()), make_move_iterator(words.end())};
 }
 
+// Split taglist, remove duplicates and convert all to lowercase
+vector<string> NormalizeTags(string_view taglist) {
```

Comment:
Since you only ever use it in for loops, maybe just return the set instead of copying it into a vector? :upside_down_face: